django_ninja_crudl/endpoints/get_one.py

```
@@ -0,0 +1,89 @@
+"""CRUDL API base class."""
```

Ruff (0.8.2)

File `django_ninja_crudl/endpoints/get_one.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)